microsoft-365/commerce/manage-billing-accounts.md

```diff
@@ -23,7 +23,7 @@ description: "Learn about billing accounts and how they're used to manage accoun
 ms.date: 03/17/2021
 ---
 
-# Understand billing accounts
+# Understand your Microsoft billing accounts
```

Comment:
@lhcole613 - Add product name to H1 for SEO?

Reply:
@SKjerland This is another one where I wasn't sure the billing accounts referred to are exclusively Microsoft 365.  So, should the H1 be: Understand your Microsoft 365 billing accounts

Reply:
@lhcole613 - Product name placement looks good. @cmcatee-MSFT - Is "Microsoft 365" correct, or should it be "Microsoft"?